webapp/src/pages/welcome/welcomePage.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+import {useIntl} from 'react-intl'
+
+import {useHistory} from 'react-router-dom'
+
+import CompassIcon from '../../widgets/icons/compassIcon'
+
+import './welcomePage.scss'
+
+type Props = {
+    workspaceID?: string,
+    userID?: string,
+}
+
+const WelcomePage = React.memo((props: Props) => {
+    const history = useHistory()
+    const intl = useIntl()
+
+    const goForward = () => {
+        localStorage.setItem('welcomePageViewed', 'true')
+        if (props.workspaceID) {
+            history.push(`/workspace/${props.workspaceID}`)
+            return
+        }
+
+        history.push('/dashboard')
+    }
+
+    if (localStorage.getItem('welcomePageViewed')) {
+        history.push('/dashboard')
+        return null
+    }
+
+    return (
+        <div className='WelcomePage'>
+            <div>
+                <h1 className='text-heading6'> {intl.formatMessage({id: 'WelcomePage.Heading', defaultMessage: 'Welcome To Boards'})} </h1>
+                <div className='text-base'> {intl.formatMessage({id: 'WelcomePage.Description', defaultMessage: 'Boards is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view'})} </div>
+
+                <div className='WelcomePage__image'>
+                    <img
+                        src='../../../../boards-welcome.png'
```

Comment:
This looks wrong, we should use something like webpack files import to solve this properly. I know now is not working fine, but this is not the solution, I think.

Reply:
Fixed. Thanks